server/conf/messages

```diff
@@ -213,6 +213,9 @@ ariaLabel.edit=Edit {0}
 # Program Summary Title
 title.programSummary=Program application summary
 
+# Error when there was an exception while submitting the application
+banner.errorSavingApplication=Error saving application
+
```

Comment:
**suggestion (blocking):** also add to messages.en-US

Reply:
@bion Do we need to have text in the other localized files as well?  We don't have translations yet, but I'm guessing it will throw an error if it tries to find the key and it doesn't exist.  Do we just put the English text in there for now?

Reply:
Answered in channel